### PR TITLE
Added file stream

### PR DESCRIPTION
**Pull request by dantwinkler**

### CODE REVIEW

File: lib/streamFactories/file.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let _ = require('lodash');
```
Comment (tonygaetani) [inline lib/streamFactories/file.js]: const

File: lib/streamFactories/index.js [hunk 2]
```diff
@@ -7,10 +7,12 @@ let logentries = require('./logentries');
 let logsene = require('./logsene');
 let prettystream = require('./prettystream');
 let logdna = require('./logdna');
+let file = require('./file');
```
Comment (tonygaetani) [inline lib/streamFactories/index.js]: const

File: test/streamFactories/file.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let expect = require('chai').expect;
```
Comment (tonygaetani) [inline test/streamFactories/file.test.js]: let -> const (3 times in this file)

**Review by tonygaetani — APPROVED** [review]
only comments are s/let/const/g and I won't let that hold this review up (but I would appreciate it!)

**Comment by tonygaetani** [comment]
let -> const 👍

**Review by jbcpollak — APPROVED** [review]

**Pull request merged**